src/components/Layouts/MainLayout/Filters/FiltersSections/AgeFilter.tsx

```diff
@@ -1,24 +1,35 @@
-import { useState, useEffect } from 'react';
-import styles from '../Filters.module.css';
+import { useEffect, Dispatch, SetStateAction } from 'react';
+
 import usersIcon from '../../../../../assets/feed/users.svg';
+import { FiltersType } from '../types';
+
+import styles from '../Filters.module.css';
 
-const AgeFilter = () => {
-  const [ageRange, setAgeRange] = useState([18, 100]);
+type Props = {
+  setFiltersData: Dispatch<SetStateAction<FiltersType>>;
+  value: number[];
+};
+
+const MIN = 18;
+const MAX = 100;
+export const AgeFilter = ({ setFiltersData, value }: Props) => {
   const updateAgeRange = (value1: number, value2: number) => {
-    setAgeRange([value1, value2]);
+    setFiltersData((state) => {
```

Comment:
```suggestion
    setFiltersData((state) => ({ ...state, age: [value1, value2] }))
```
in all places where you update state

Reply:
agreed